completion/openqa-cli-completion.bash

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
I would still prefer a more generic directory name. We don't want to create a top level subfolder for every kind of feature we add.
I suggest either `share/completion` , or we use the existing `contrib` directory, so `contrib/completion`.
Other opinions?

Reply:
`contrib/completion` seems like a nice place

Reply:
done